packages/footer/components/SocialMedia.tsx

```diff
@@ -21,12 +22,21 @@ export interface SocialMediaAccounts {
 
 export const SocialMediaSection = memo(function SocialMediaSection({
    accounts,
+   repairUrl,
 }: {
    accounts: SocialMediaAccounts;
+   repairUrl?: string;
 }) {
    if (!accounts) {
       return null;
    }
+
+   let showRepairEU = false;
+
+   if (repairUrl && repairUrl.includes('eu')) {
```

Comment:
We can just move this check inline with the `icon={showRepairEU ? RepairEULogo : RepairOrgLogo}` and get rid of this unneeded variable.